sdk/java/pom.xml

```diff
@@ -85,6 +85,14 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <release>11</release>
```

Comment:
Somewhat of a question whether SDK should build for 8 or 11 perhaps. It was implicitly 11 since that become default in the root, so I preserved that, but I can also change it now if there is a desire to. Needs a small code change in tests I think.

Reply:
haha. @khorshuheng just ran into this during our upgrade today.

Reply:
I'll take that as a real-world indicator that yes, we should keep it Java 8-compatible. Pushed this change.